app/views/smart_proxies/show.html.erb

```diff
@@ -69,5 +75,10 @@
         </div>
       </div>
     <% end %>
+    <% sorted_pagelets_at(:tabbed_menu).each do |tab| %>
+      <div id="<%= tab.snake_name %>" class="tab-pane">
+        <%= render tab.partial, { :smart_proxy => @smart_proxy, :pagelet => tab }%>
```

Comment:
Yeah, I like you pass pagelets in. Can you do the same for all extension points?

Best approach would be to create `render_pagelet` helper that would do this for us accepting all variables that must be sent in.